<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>20</width>
    <height>20</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Weather Judgment Hub</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="log_API_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>651</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="manual_tab">
     <attribute name="title">
      <string>Manual</string>
     </attribute>
     <widget class="QLabel" name="message">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>731</width>
        <height>191</height>
       </rect>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>en attente de DATA .</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Température Cel :</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="input_vent">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Pression atmosphérique en pascal :</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="input_temp">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="input_atmo">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="maximum">
       <double>10000.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Vitesse vent en noeud :</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="api_tab">
     <attribute name="title">
      <string>API</string>
     </attribute>
     <widget class="QLineEdit" name="input_api_keys">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_ville">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Votre clée API</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_api">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>231</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>En attent de clée API</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Votre ville :</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>431</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="API_Start_B">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>121</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Lancer l'appel API</string>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>431</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_API_vent">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>50</y>
        <width>311</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>vitesse de vent en noeud : NO DATA</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_API_atmo">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>80</y>
        <width>351</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>pression atmospherique en pascals : NO DATA</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_API_temp">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>110</y>
        <width>281</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>température en °C : NO DATA</string>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>0</y>
        <width>20</width>
        <height>221</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>30</y>
        <width>431</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="TITEL">
      <property name="geometry">
       <rect>
        <x>461</x>
        <y>10</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Données envoyées par l’API</string>
      </property>
     </widget>
     <widget class="QLabel" name="message_API">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>781</width>
        <height>191</height>
       </rect>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">/* Style du QGroupBox (Conteneur) */
QGroupBox {
    border: 3px solid #1abc9c;
    border-radius: 10px;
    margin-top: 15px; 
    background-color: #ecf0f1;
}

/* Style du titre du GroupBox */
QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left; /* Haut à gauche */
    padding: 0 10px;
    color: #1abc9c;
    font-size: 14px;
    font-weight: bold;
    background-color: #ecf0f1;
}

/* Style des Labels DANS le GroupBox */
QGroupBox QLabel {
    color: #2c3e50;
    font-family: &quot;Arial&quot;, sans-serif;
    font-size: 12px;
    padding: 5px;
}

/* Style d'un QLabel spécifique (doit avoir objectName=&quot;label_etat_important&quot;) */
QGroupBox #label_etat_important {
    color: white;
    background-color: #e74c3c; /* Rouge pour l'état */
    border: none;
    border-radius: 5px;
    padding: 8px 15px; /* Plus d'espace pour l'effet de bouton/badge */
    font-weight: bold;
}</string>
      </property>
      <property name="text">
       <string>en attente de DATA .</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="log_API">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>470</y>
        <width>741</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Pas d'erreur pour la moment</string>
      </property>
     </widget>
     <widget class="QLabel" name="City_long">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>140</y>
        <width>281</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>longitude de la ville :</string>
      </property>
     </widget>
     <widget class="QLabel" name="City_lat">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>170</y>
        <width>281</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>latitude de la ville :</string>
      </property>
     </widget>
     <widget class="QLabel" name="deg_vent">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>180</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    /* Augmente la taille de la police */
    font-size: 36px; 
    
    /* Autres options de style (facultatif) */
    color: #34495e; /* Couleur de texte bleu marine */
    font-weight: bold; /* Texte en gras */
}</string>
      </property>
      <property name="text">
       <string>←</string>
      </property>
     </widget>
     <widget class="QLabel" name="City_lat_2">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>200</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>orientation du vent :</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
